packages/react-components/react-datepicker-compat/src/index.ts

```diff
@@ -15,31 +8,3 @@ export {
   useDatePickerStyles_unstable,
 } from './DatePicker';
 export type { DatePickerErrorType, DatePickerProps, DatePickerValidationResultData } from './DatePicker';
-
-export {
-  DAYS_IN_WEEK,
-  DateRangeType,
-  DayOfWeek,
-  FirstWeekOfYear,
-  MonthOfYear,
-  TimeConstants,
-  addDays,
-  addMonths,
-  addWeeks,
-  addYears,
-  compareDatePart,
-  compareDates,
-  getDatePartHashValue,
-  getDateRangeArray,
-  getEndDateOfWeek,
-  getMonthEnd,
-  getMonthStart,
-  getStartDateOfWeek,
-  getWeekNumber,
-  getWeekNumbersInMonth,
-  getYearEnd,
-  getYearStart,
-  isInDateRangeArray,
-  setMonth,
-} from './utils';
-export type { CalendarStrings, DateFormatting, DateGridStrings } from './utils';
```

Comment:
Completely removing these exports is technically a breaking change, should these be re-exported from `react-calendar-compat` instead?

Reply:
That's something I was battling. I did make the change a minor(braking for compat) since I'm removing the exports. What I was thinking though is that by adding datepicker to your dependencies you're pulling calendar, so explicitly adding the calendar dependency shouldn't be an issue. This can keep the exports of datepicker simpler. Although I see the point of not wanting to do so and in that case would be better to re-export from calendar. I'm open to make that change if you think it's more ergonomic to do it that way.

Reply:
Yeah not re-exporting is definitely more ergonomic. I do see your point regarding the calendar dependency being pulled in through datepicker but it would still initially break folks since they would need to update their import statements to pull from calendar instead of datepicker. Since `minor` changes in compat packages are expected to be breaking like you said, I'm ok with keeping this PR as is but I would probably update the change file description to include the removal of react-calendar-compat related exports from react-datepicker-compat

Reply:
sounds good, I'll keep the change for removing the exports and update the change file